exercises/05/readme.md

```diff
@@ -171,6 +171,8 @@ Put the new entity through its paces by performing some OData Create operations
 
 An OData Create operation (request and response) to insert a new order looks in raw form like this:
 
+> Note: do not try to execute the following lines somewhere, this is just an illustration of the HTTP Request and HTTP Response
```

Comment:
The "somewhere" sounds very vague, maybe we can find a better term

Reply:
fully agree. replaced somewhere with "at all".

Reply:
I do think that while this note has good intentions, it sounds like it fits into the same ballpark as "don't forget to press Enter". There are no directions here to do anything, and the descriptive text is surely context enough for the reader to realise it's an illustration.

Reply:
again and as usual, you are totally right, DJ. I just had so many people being confused with what to do here and it would make things as a instructor/expert (us) a little bit easier ;-)

Reply:
Agree, but I would suggest this is more "teach a person to fish, rather than give them a fish". Help them to practise reading instructions :-)